### PR TITLE
Refs #31003 -- Moved note about return value of QuerySet.bulk_create() to the first paragraph.

### PR DESCRIPTION
**Pull request by claudep**

### CODE REVIEW

File: docs/ref/models/querysets.txt [hunk 1]
```diff
@@ -2157,9 +2157,9 @@ exists in the database, an :exc:`~django.db.IntegrityError` is raised.
 
 This method inserts the provided list of objects into the database in an
 efficient manner (generally only 1 query, no matter how many objects there
-are)::
+are), and returns ``objs`` as cast to a list, in the same order as provided.
 
-    >>> Entry.objects.bulk_create([
+    >>> objs = Entry.objects.bulk_create([
```
Comment (felixxm) [inline docs/ref/models/querysets.txt]:
```suggestion
    >>> Entry.objects.bulk_create([
```

**Review by felixxm — COMMENTED** [review]
@claudep Thanks :+1: I pushed minor edits.

File: docs/ref/models/querysets.txt [hunk 2]
```diff
@@ -2157,9 +2157,9 @@ exists in the database, an :exc:`~django.db.IntegrityError` is raised.
 
 This method inserts the provided list of objects into the database in an
 efficient manner (generally only 1 query, no matter how many objects there
-are)::
+are), and returns ``objs`` as cast to a list, in the same order as provided.
```
[the remark this answers is not in the document]
Comment (claudep) [reply docs/ref/models/querysets.txt]: Or maybe simply `…and returns created objects as a list, in the same…`?

[the remark this answers is not in the document]
Comment (felixxm) [reply docs/ref/models/querysets.txt]: Updated.

**Comment by claudep** [comment]
Thanks for the improvement.

**Pull request merged**